src/lib/ExecutionLib.sol

```diff
@@ -22,25 +22,41 @@ library ExecutionLib {
          */
         // solhint-disable-next-line no-inline-assembly
         assembly ("memory-safe") {
```

Comment:
@zeroknots I fixed `decodeSingle` but `decodeBatch` does not work properly yet (hence the batch test is failing)

Reply:
fixed in

https://github.com/erc7579/uMSA/pull/8/commits/9e7e69d91d44b1dc904f441046cbfe6d3eb16dc2